PodcastGenerator/setup/createconf.php

```diff
@@ -105,6 +105,11 @@ function createconf($username, $password) {
 
 \$users_json = \"{\\\"".$username."\\\": \\\"".str_replace("\$", "\\\$", $password)."\\\"}\";
 
+#####################
+# WebSub
+
+\$websub_server = \"https://pubsubhubbub.appspot.com/\";
```

Comment:
I don't we should pre-set this by default. This isn't really privacy friendly.

Reply:
Done in 913846d